docs/resources/aom_alarm_rule.md

```diff
@@ -0,0 +1,122 @@
+---
+subcategory: "Application Operations Management (AOM)"
+---
+
+# huaweicloud_aom_alarm_rule
+
+Manages a AOM alarm rule resource within HuaweiCloud.
+
+## Example Usage
+
+### Basic example
+
+```hcl
+resource "huaweicloud_aom_alarm_rule" "alarm_rule" {  
+  alarm_name           = "test-rule"
+  alarm_level          = 3
+  alarm_description    = "test rule"
+
+  namespace   = "PAAS.NODE"
+  metric_name = "cupUsage"
+
+  dimensions {
+    name  = "hostID"
+    value = var.instance_id
+  }
+
+  comparison_operator = ">="
+  period              = 60000
+  statistic           = "average"
+  threshold           = 3
+  unit                = "Percent"
+  evaluation_periods  = 2
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String, ForceNew) The region in which to create the alarm rule resource. If omitted, the
+  provider-level region will be used. Changing this creates a new resource.
+
+* `alarm_name` - (Required, String, ForceNew) Specifies the name of an alarm rule. The value can be a string of 1 to 100
+  characters that can consist of letters, digits, underscores (_), hyphens (-) and chinese characters,
+  and it must start and end with letters, digits or chinese characters. Changing this creates a new resource.
+
+* `alarm_description` - (Optional, String) Specifies the description of the alarm rule.
+ The value can be a string of 0 to 1000 characters.
+
+* `alarm_level` - (Optional, Int) Specifies the alarm severity. The value can be **1**, **2**, **3** or **4**,
+  which indicates *critical*, *major*, *minor*, and *informational*, respectively.
+  The default value is **2**.
+
+* `alarm_actions` - (Optional, List, ForceNew) Specifies the action triggered by an alarm. This is a list of strings.
+  Changing this creates a new resource.
+
+* `alarm_action_enabled` - (Optional, Bool, ForceNew) Specifies whether to enable the action to be triggered by an alarm.
+  The default value is true. Changing this creates a new resource.
+
+* `ok_actions` - (Optional, List, ForceNew) Specifies the action triggered by the clearing of an alarm.
+  This is a list of strings. Changing this creates a new resource.
+
+* `insufficient_data_actions` - (Optional, List, ForceNew) Specifies the action triggered when the data is not enough.
+  This is a list of strings. Changing this creates a new resource.
+
+* `metric_name` - (Required, string, ForceNew) Specifies the alarm metric name. Changing this creates a new resource.
+
+* `namespace` - (Required, String, ForceNew) Specifies the alarm namespace. Changing this creates a new resource.
+
+* `period` - (Required, Int) Specifies the alarm checking period in milliseconds.
+  The value can be **60000**, **300000**, **900000** and **3600000**.
```

Comment:
suggest to mark these numbers with \' and add the comma to separate them:
`60,000`
`300,000`
`900,000`
`3,600,000`

Reply:
fixed